src/EnergyPlus/HVACInterfaceManager.cc

```diff
@@ -798,15 +798,14 @@ void ManageSingleCommonPipe(EnergyPlusData &state,
     int NodeNumSecIn = state.dataPlnt->PlantLoop(LoopNum).LoopSide(DataPlant::LoopSideLocation::Demand).NodeNumIn;
     int NodeNumSecOut = state.dataPlnt->PlantLoop(LoopNum).LoopSide(DataPlant::LoopSideLocation::Demand).NodeNumOut;
 
-    auto &MyEnvrnFlag = PlantCommonPipe(LoopNum).MyEnvrnFlag;
-    if (MyEnvrnFlag && state.dataGlobal->BeginEnvrnFlag) {
+    if (PlantCommonPipe(LoopNum).MyEnvrnFlag && state.dataGlobal->BeginEnvrnFlag) {
```

Comment:
PlantCommonPipe(LoopNum) use shortcut?

Reply:
Creating a reference to PlantCommonPipe(LoopNum) does not speed up code execution, it makes code more readable. In this case the code reads fine as it is. If I don't see an improvement in code readability, I think twice about using that type of reference.

Reply:
I should do that, but not in this PR. This is an idiom/pattern clean up pass across many files. If I combine this kind of change with it, the diffs and conflicts will be too big. This kind of change is good to do one or two files at a time. FWIW I do have another branch `FixBadAuto2` in which I got carried away and did some other refactoring in addition to `auto` work. I have spent almost as much time resolving conflicts on that branch as creating it in the first place.

Reply:
yeah, maybe it should not be in this PR.